app/src/main/res/layout/fragment_add.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
i mean, the gui could be much more, such as deletion, cancellation (when adding a new), specifying what the number you are choosing is, allowing hours/weeks, and stuff... i guess you dont wanna do it now, so take everything as a suggestion.